libimage/layer_tree.go

```diff
@@ -22,6 +23,8 @@ type layerTree struct {
 	// emptyImages do not have any top-layer so we cannot create a
 	// *layerNode for them.
 	emptyImages []*Image
+
+	manifestDigests map[digest.Digest]bool
```

Comment:
Please document keys/values. And if it is effectively a set, make the values `struct{}`.

Reply:
Not sure what you are after here.  This is just a mapping of digests that are used in the manifest lists in containers/storage.